### DISCUSSION TITLE
GUI crashed after a day or two of the plotting

### DISCUSSION THREAD

**Discussion by abedae** [Chia-Network/chia-blockchain#5329] [Support]
I'm plotting with the GUI (Win 10 pro)

  This is the fourth time crashed  I've lost over 20 plots due to this problem without knowing the reason

 This is the last information in the debug file
`
2021-05-17T09: 23: 54.788 full_node full_node_server: WARNING Trying to ban localhost for 10, but will not ban
2021-05-17T09: 23: 54.789 full_node asyncio: ERROR Task exception was never retrieved
future: <Task finished coro = <ChiaServer.incoming_api_task. <locals> .api_call () done, defined at chia \ server \ server.py: 482> exception = CancelledError ()>
concurrent.futures._base.CancelledError
2021-05-17T09: 23: 54.790 full_node full_node_server: ERROR Exception: <class 'concurrent.futures._base.CancelledError'>, closing connection {'host': '127.0.0.1', 'port': 8449}. Traceback (most recent call last):
  File "chia \ server \ server.py", line 531, in api_call
  File "asyncio \ tasks.py", line 435, in wait_for
concurrent.futures._base.CancelledError `

**Comment by steven-b89** [comment]
Plotting on Windows with Powershell, no issues at all.

Open Powershell and do following:

cd ~\AppData\Local\chia-blockchain\app-1.1.5\resources\app.asar.unpacked\daemon\
.\chia plots create -k 32 -b <MEMORY AMOUNT> -r CPU THREADS -n NUMBER OF PLOTS -t TEMP LOCATION -d FINAL DESTINATION

If you are running on a single system and using GUI you can close gui and give following command:
.\chia start farmer

More information can be found here:
https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference

Running with Powershell it is harder to track the progress but on link below there is a section where every phase is explained and give you a good indication at where you're currently at while plotting:
https://github.com/Chia-Network/chia-blockchain/wiki/Beginners-Guide

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
